app/assets/javascripts/discourse/app/templates/components/category-logo.hbs

```diff
@@ -0,0 +1,10 @@
+<div class="aspect-image">
```

Comment:
Maybe we should change this to 
```suggestion
<div class="category-logo aspect-image">
```

What do you think? It would make for more reasonable style names in the stylesheets.

Reply:
I don't quite get the "Suggested change". Do you mean adding `category-logo` back with a better name or renaming `aspect-image` something more useful? `category-logo-container` maybe? I don't think we can get much more specific than that.